javascript-practice/src/pages/dashboard.html

```diff
@@ -27,23 +31,27 @@ <h2 id="fullName" class="header-dashboard-profile-name"></h2>
               </div>
               <p id="role" class="header-dashboard-profile-sub-name"></p>
             </div>
-            <div class="flex-column">
-              <img
-                id="arrow"
-                class="arrow"
-                src="../assets/icons/arrow.svg"
-                alt="arrow"
-              />
-              <span
-                id="logout"
-                class="header-dashboard-profile-sub-name logout"
-              >
-                Logout
-              </span>
-            </div>
+          </div>
+          <div class="flex-column">
+            <img
+              id="arrow"
+              class="arrow"
+              src="../assets/icons/arrow.svg"
+              alt="arrow"
+            />
+            <span id="logout" class="header-dashboard-profile-sub-name logout">
```

Comment:
header-dashboard-profile-sub-name??? What related between logout button and this class